src/pages/developers/overview.mdx

```diff
@@ -4,10 +4,10 @@ At a high level, ZetaChain is a Proof of Stake (PoS) blockchain developed using
 the Cosmos SDK and Comet BFT consensus engine. As a result, ZetaChain enjoys
 fast block time and instant finality.
 
-ZetaChain hosts an Ethereum Virtual Machine (EVM) compatible execution layer. In
-addition to supporting all EVM features and standard interactions (such as
-contract creation, contract interaction, and contract composition), ZetaChain's
-EVM distinguishing characteristics include:
+ZetaChain hosts an Ethereum Virtual Machine (EVM) compatible execution layer
+called the Universal EVM. In addition to supporting all EVM features and
+standard interactions (such as contract creation, contract interaction, and
+contract composition), ZetaChain's EVM distinguishing characteristics include:
```

Comment:
Could we mention Chain Abstraction Framework here?

Reply:
7730531